docs/log_metric_callback.rst

```diff
@@ -6,14 +6,15 @@
 ***********************
 Metric Logging Callback
 ***********************
-
-With the director-based workflow, you can use custom metric logging function for each task via Python\*\  API or command line interface. This function calls on the aggregator node.
-
+You can use custom metric logging function for each task via Python\*\  API or command line interface. This function calls on the aggregator node.
 
```

Comment:
Can we come up with a code snippet for the Metric Logging Callback? A small example could help.

Reply:
+1 . An example would really help here